tests/initial_conditions/test_initial_conditions.py

```diff
@@ -39,9 +41,37 @@ def test_get_data_source(
             pytest.skip("Could not initialize client")
 
     ds = initial_conditions.get_data_source(
-        n_history=0,
         channel_names=["t850", "t2m"],
-        grid=schema.Grid.grid_721x1440,
         initial_condition_source=source,
     )
     assert isinstance(ds, DataSource)
+
+
+@pytest.mark.parametrize("n", [1, 2])
```

Comment:
Can this tested for n=0 too?

Reply:
note that n=n_history_levels = 1+n_history. I did this so that x.shape[1] == n_history_levels. so n=0 corresponds to an tensor with x.shape[1] = 0, which doesn't make sense.